opentelemetry-operator/chart/templates/logs-collector.yaml

```diff
@@ -8,7 +8,7 @@ kind: OpenTelemetryCollector
 metadata:
   name: logs 
 spec:
-  image: ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib:0.104.0
+  image: {{ index .Values "opentelemetry-operator" "manager" "collectorImage" "repository" }}:{{ index .Values "opentelemetry-operator" "manager" "collectorImage" "tag" }}
```

Comment:
Note: You do not always have to use the index function because there is a `-` in the name of the dependent chart. You can also define an alias for it. https://github.com/cloudoperators/greenhouse-extensions/blob/c31b80ca5750d21b18842d7614d925ea51ad7224/kube-monitoring/charts/Chart.yaml#L20